src/ChartContainer/index.js

```diff
@@ -64,13 +70,30 @@ function ChartContainer({
         justify="space-between"
```

Comment:
Make buttons always align top for mobile i.e. `alignItems="flex-start"`

![Screenshot from 2019-08-21 17-14-02](https://user-images.githubusercontent.com/1779590/63439679-2ab10380-c437-11e9-9150-43b19b896209.png)